include/spirv-tools/optimizer.hpp

```diff
@@ -521,8 +521,14 @@ Optimizer::PassToken CreateDeadInsertElimPass();
 // interface are considered live and are not eliminated. This mode is needed
 // by GPU-Assisted validation instrumentation, where a change in the interface
 // is not allowed.
+//
+// If |remove_outputs| is true, allow outputs to be removed from the interface.
+// This is only safe if the caller knows that there is no corresponding input
+// variable in the following shader. It is false by default.
```

Comment:
it's false by default on the pass constructor, here we still have is mandatory, is that expeted?

Reply:
> ...here we still have is mandatory...

Not sure what you mean when you say it is mandatory here. It is ok if the following shader had an unused interface variable removed, but we do not remove it in this shader. The Vulkan spec says that is ok. It is strictly an optimization to remove it. It is not mandatory.

Reply:
I meant the comment says it's false by default, but you require the parameter without default value. (the underlying pass constructor takes has the default value set to false, not this one)

Reply:
Ah. I see. Yeah. Not sure why I did it this way. Technically, there were two overloads so it wasn't really mandatory, but I changed to a more standard and simpler way to do this.

Fixed.